expression/builtin.go

```diff
@@ -413,6 +413,7 @@ var funcs = map[string]functionClass{
 	ast.Year:             &yearFunctionClass{baseFunctionClass{ast.Year, 1, 1}},
 	ast.YearWeek:         &yearWeekFunctionClass{baseFunctionClass{ast.YearWeek, 1, 2}},
 	ast.LastDay:          &lastDayFunctionClass{baseFunctionClass{ast.LastDay, 1, 1}},
+	ast.Tso:              &tsoFunctionClass{baseFunctionClass{ast.Tso, 1, 1}},
```

Comment:
We'd better change ast.Tso to ast.TidbParseTso

Reply:
ast.Tso already merged in parser master

Reply:
We can push another PR to change the name